lerna.json

```diff
@@ -3,5 +3,7 @@
   "packages": [
     "packages/*"
   ],
-  "version": "0.0.0"
+  "version": "2.0.0-alpha",
+  "npmClient": "yarn",
+  "useWorkspaces": true
```

Comment:
This is to avoid dependencies duplication. https://yarnpkg.com/lang/en/docs/workspaces/